content/en/releases/patch-releases.md

```diff
@@ -81,6 +81,8 @@ releases may also occur in between these.
 | July 2023             | 2023-07-14           | 2023-07-19  |
 | August 2023           | 2023-08-04           | 2023-08-09  |
 | September 2023        | 2023-09-08           | 2023-09-13  |
+| October 2023          | 2023-10-06           | 2023-10-11  |
+| November 2023         | 2023-11-10           | 2023-11-15  |
```

Comment:
Do we want to add +1 week because of KubeCon? We could also consider skipping the December releases then.

Reply:
+1

Reply:
I think it's worth to mention that we're talking about an US holiday, while the community strives to be as less US centric as possible. Nevertheless our Google Build Admin helping us to cut deb and rpm packages is based in the US, so we **have** to consider that.

How about using the first week of December because of that?

Reply:
Yes, that's what I thought as well. As mentioned in the PR description:

> I think we could look at 2023-12-01 as cherry-pick deadline and 2023-12-06 as the target date.

I think that's probably the best plan.

Reply:
I adjusted the calendar and added a note about this, PTAL.